javafx-archetype-simple/src/main/resources/archetype-resources/pom.xml

```diff
@@ -23,7 +30,11 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.0</version>
                 <configuration>
+#if ("8" == ${javafx-version})
+                    <excludes>module-info.java</excludes>
+#else
```

Comment:
If the project targets 8, and it is not intended to run on 11+, `module-info.java` should be excluded from the source code that the archetype generates. 
The JavaFX plugin can compile the project too, but doesn't allow the `excludes` tag (at least for now).

Reply:
See the comment in https://github.com/openjfx/javafx-maven-archetypes/pull/8 for this.